src/components/__tests__/collapsible.test.tsx

```diff
@@ -7,12 +7,12 @@ const openedText = "Close me"
 const closedText = "Open me"
 const content = "I am content"
 
-const renderWrapper = ({ isInitiallyCollapsed = true, onChange = null } = {}) =>
```

Comment:
so from my understanding we will drop the isInitiallyCollapsed? aren't we using this in filters? 🤔  maybe you can explain a bit how forceCollapse is replacing the isInitiallyCollapsed cuz i am a bit confused

Reply:
No, it will be replaced by `forceCollapse`. The difference is that when `forceCollapse` would change it would force the collapsible state to update. Whereas `isInitiallyCollapsed` was only responsible for setting the initial state.

I tried rolling with additional property and keeping it in sync with `isInitiallyCollapsed`, but it made the code insanely complex and hart to follow and allowed for some weird edge cases that would result in an impossible state (`forceCollapse: true` &`isInitiallyCollapsed: false` 🤯 )

Reply:
aaa gotcha now. makes sense to roll just with forceCollapse. thanks for explaining it.